db/README.md

```diff
@@ -29,21 +29,20 @@ General advice:
 #### which objects aren't in a good state?
 ```ruby
 # example AR query
-[25] pry(main)> PreservedCopy.joins(:preserved_object, :endpoint, :status).where("statuses.status_text != 'ok'").order('statuses.status_text').pluck('statuses.status_text, preserved_objects.druid, endpoints.storage_location')
+[25] pry(main)> PreservedCopy.joins(:preserved_object, :endpoint).where.not(status: :ok).order('preserved_copies.status asc, endpoints.storage_location asc').pluck(:status, :storage_location, :druid)
```

Comment:
nice catch, changing the README

Reply:
as a bonus, i think the AR queries are more readable with the schema change from #228, and the generated sql queries are also simpler and more readable.